### ISSUE TITLE
-f or --file not working

### ISSUE THREAD

**Issue by adamjerickson** [HamburgChimps/apple-notes-liberator#49]
I am trying to extract notes from my daughter's ipad. I recovered the NoteStore file and am trying to run this on my mac which also has Notes. But my local notes are not the desired NoteStore. I am using thie `-f` and `--file` option, but it is always only extracting the default notes file.

I have them in a folder on my desktop. I am in that directory. This is the command I'm running.

```
apple-notes-liberator --file=NoteStore.sqlite --json
```

have also used 
```
apple-notes-liberator --file=./NoteStore.sqlite --json
```

Have tried -f with above paths with same result. 
Have also tried --markdown output with same result.

Version is
```
xxxxxx@xxxxxx LizRecoveredNotes % apple-notes-liberator --version                     
2.2.3
```

**Comment by adamjerickson** [comment]
```
xxxxxx@xxxxxx LizRecoveredNotes % ls -l
total 81152
-rwxrwxrwx@ 1 adamerickson  staff  41541632 Jul  2 21:33 NoteStore.sqlite
-rwxrwxrwx@ 1 adamerickson  staff       261 Jan 20  2021 NotesIndexerState-Legacy
-rwxrwxrwx@ 1 adamerickson  staff       261 Jul  2 21:33 NotesIndexerState-Modern
-rw-r--r--@ 1 adamerickson  staff         0 Jul  2 23:23 SharingExtensionDeviceUUID
-rw-r--r--@ 1 adamerickson  staff         0 Jul  2 23:23 SharingExtensionWriteLock
-rw-r--r--@ 1 adamerickson  staff         0 Jul  2 23:23 com.apple.notes.databaseopen.lock
```

**Comment by alexkolson** [comment]
Hey @adamjerickson! Reviewing the code I found a glaring mistake - even though I was checking to see if a file existed at the path provided unter the `-f` or `--file` flag, I was always taking the hardcoded notestore path. 🙄 I'm not sure if it was always this way or if I broke it along the way. I've just released [v2.2.4](https://github.com/HamburgChimps/apple-notes-liberator/releases/tag/v2.2.4) that should fix this and actually **use** the file provided via the command line `-f` or `--file` option. Could you give it a try and let me know if it works for you?

**Comment by alexkolson** [comment]
Hello @adamjerickson I am going to close this, if the issue still persists with the latest version please do not hesitate to open a new issue.
